packages/aws-cdk-lib/aws-cloudwatch/lib/graph.ts

```diff
@@ -71,6 +71,21 @@ export interface YAxisProps {
   readonly showUnits?: boolean;
 }
 
+/**
+ * Properties for lables shown on the graph.
+ */
+export interface LabelProps {
+  /**
+   * Whether to display labels on chart.
+   *
+   * If false, no labels are shown directly on the pie chart;
+   * if true, each segment of the pie chat is labelled with the percentage contribution.
+   *
+   * @default false
```

Comment:
I'm not sure the default is always false, I think this changes depending on the chart type, see [docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/CloudWatch-Dashboard-Body-Structure.html#CloudWatch-Dashboard-Properties-Variables-Structure)

I would also remove the reference to pie charts, since I assume this property can be used for all chart types